nsdb-cluster/src/test/scala/io/radicalbit/nsdb/cluster/coordinator/AbstractReadCoordinatorSpec.scala

```diff
@@ -40,6 +40,7 @@ abstract class AbstractReadCoordinatorSpec
           .load()
           .withValue("akka.actor.provider", ConfigValueFactory.fromAnyRef("local"))
           .withValue("nsdb.sharding.interval", ConfigValueFactory.fromAnyRef("5s"))
+          .withValue("akka.log-dead-letters-during-shutdown", ConfigValueFactory.fromAnyRef("off"))
```

Comment:
I guess we can disable this config for all the tests

Reply:
what about doing it with a ad-hoc pr?

Reply:
Makes sense. Agreed